### PR TITLE
change capitalization and phrasing of some sentences on wallet page

### PR DESCRIPTION
**Pull request by lemonase**
These are not a huge deal, but just some things I noticed while reading through.

## Description

Changed capitalization of OSes for sake of consistency (MacOS was the only one capitalized). The rest of the changes are just word choice and sentence structure (for clarity). 

Thanks!

### CODE REVIEW

File: src/intl/en/page-wallets.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
   "page-wallets-desc-3": "Your wallet is only a tool for managing your Ethereum account. That means you can swap wallet providers at any time. Many wallets also let you manage several Ethereum accounts from one application.",
   "page-wallets-desc-4": "That's because wallets don't have custody of your funds, you do. They're just a tool for managing what's really yours.",
   "page-wallets-description": "Ethereum wallets are applications that let you interact with your Ethereum account. Think of it like an internet banking app – without the bank. Your wallet lets you read your balance, send transactions and connect to applications.",
-  "page-wallets-desktop": "Desktop applications if you prefer to manage your funds via MacOS, windows or linux",
-  "page-wallets-ethereum-wallet": "A wallet is a product that lets you manage your Ethereum account, like view your account balance, send transactions, and more.",
+  "page-wallets-desktop": "Desktop applications if you prefer to manage your funds via MacOS, Windows or Linux",
```
Comment (wackerow) [inline src/intl/en/page-wallets.json:20]: while we're in here, I would suggest make it `macOS`

**Review by wackerow — CHANGES_REQUESTED** [review]
@lemonase Just made one comment, otherwise looks good to me =)

File: src/intl/en/page-wallets.json (hunk 1 above)
Comment (lemonase) [reply src/intl/en/page-wallets.json]: Yep that's good with me. Had to check on Apple's website because I don't have a mac haha.

**Review by lemonase — COMMENTED** [review]
Caps change

**Review by lemonase — COMMENTED** [review]
@wackerow  So here's the actual commit for that change (still getting accustomed to the PR flow on GitHub)

**Review by wackerow — APPROVED** [review]
@lemonase 👍 thanks!

**Comment by wackerow** [comment]
Sorry for delays on responding here. No worries on the git stuff 🙂 feel free to ping us in Discord if you ever have issues. Pulling this in now

**Comment by wackerow** [comment]
@all-contributors please add @lemonase for content patch

**Pull request merged**